tests/Oryx.Integration.Tests/Fixtures/SqlServerDbContainerFixture.cs

```diff
@@ -1,94 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-// --------------------------------------------------------------------------------------------
-
-using System;
-using System.Collections.Generic;
-using Microsoft.Oryx.Tests.Common;
-using Polly;
-using Xunit;
-
-namespace Microsoft.Oryx.Integration.Tests.Fixtures
-{
-    public class SqlServerDbContainerFixture : DbContainerFixtureBase
```

Comment:
I am not convinced that we need to have this kind of support. It is a one time setup and if anyone does the changes incorrectly the tests would fail and we go and fix it on Azure and forget it. As of now I do not think we will see the need to modify the data anytime soon.

Reply:
It's not about modifying the data - it's about "self healing".
Imagine we need to re-create that server in a few months - not too unlikely - wouldn't it be nice if we'll be able to just update the connection string in our Key Vault, and have the test data repopulated without any intervention?